src/main/java/org/cats/stock/controller/ReceiptController.java

```diff
@@ -19,6 +19,7 @@
  * Created by alexander on 1/4/17.
  */
 
+@CrossOrigin(origins = "*", allowedHeaders = "Origin, X-Requested-With, Content-Type, Accept")
```

Comment:
@asuraphel I'm assuming that we will be restricting this to limit requests from cats-web only.